CONTRIBUTING.md

```diff
@@ -174,15 +174,15 @@ Next, let's provide some extra styling for the example element. Create a new CSS
 
 ### Updating the metadata
 
-Next, you need to tell the page generator about your new page and its dependencies. To do this, open up the `site.json` file in the root of the project folder. Under `pages`, find an existing entry with a `type` of `css`.
+Next, you need to tell the page generator about your new page and its dependencies. To do this, open up the  To do this, open up `meta.json` at current folder (i.e. at "live-examples/css-examples/backgrounds-and-borders/meta.json").
```

Comment:
You are accidentally repeating `To do this, open up`

> open up `meta.json` at current folder (i.e. at "live-examples/css-examples/backgrounds-and-borders/meta.json").

open up the `meta.json` file in the current folder (i.e. "live-examples/css-examples/backgrounds-and-borders/meta.json").